api/api/spec/spec.yaml

```diff
@@ -1453,10 +1453,6 @@ components:
 
     AgentGroup:
       type: object
-      required:
-      - count
-      - name
-      - configSum
```

Comment:
I simply removed them all since the other endpoints with required fields are not in the spec. Plus, when using distinct it won't be in the response and the integration tests are validating that it's always present